test_scripts/Defects/5_0/1903_Media_app_is_not_activated_during_active_embedded_navigation.lua

```diff
@@ -0,0 +1,69 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1903
+--
+-- Description:
+-- Media app is not activated during active embedded navigation.
+-- Precondition:
+-- SDL and HMI are started.
+-- In case:
+-- 1) Media app in LIMITED and AUDIBLE due to active embedded navigation.
+-- 2) SDL receives SDL.ActivateApp (<appID_of_media_app>) from HMI.
+-- Expected result:
+-- 1) SDL must respond SDL.ActivateApp (SUCCESS) to HMI
+--    send OnHMIStatus (FULL, AUDIBLE) to mobile app (embedded navigation is still active).
+-- Actual result:
+-- SDL does not set required HMILevel and audioStreamingState.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('user_modules/sequences/actions')
+local runner = require('user_modules/script_runner')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+config.application1.registerAppInterfaceParams.appHMIType = { "MEDIA" }
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+--[[ Local Functions ]]
+local function embeddedNavigation()
+    common.getHMIConnection():SendNotification("BasicCommunication.OnEventChanged",
+    {eventName = "EMBEDDED_NAVI", isActive = true})
+
+    common.getMobileSession():ExpectNotification("OnHMIStatus",
+    {hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE", systemContext = "MAIN"})
+end
+
+local function activateMediaApp()
+    local requestId = common.getHMIConnection():SendRequest("SDL.ActivateApp", { appID = common.getHMIAppId() })
+    EXPECT_HMIRESPONSE(requestId)
+    :Do(function()
+        common.getHMIConnection():SendNotification("BasicCommunication.OnEventChanged",
+        {eventName = "EMBEDDED_NAVI", isActive = true})
+    end)
+    common.getMobileSession():ExpectNotification("OnHMIStatus")
+    :ValidIf(function(_, data)
```

Comment:
@HSavynetska Instead of custom `ValidIf` it's better to define expected values directly in expectation